src/dotnet/commands/dotnet-restore/NuGet3.cs

```diff
@@ -10,13 +10,13 @@ namespace Microsoft.DotNet.Tools.Restore
 {
     internal static class NuGet3
     {
-        public static int Restore(IEnumerable<string> args, bool quiet)
+        public static int Restore(IEnumerable<string> args)
         {
             var prefixArgs = new List<string>();
-            if (quiet)
+            if (!args.Any(s => s.Equals("--verbosity", StringComparison.OrdinalIgnoreCase) || s.Equals("-v", StringComparison.OrdinalIgnoreCase)))
             {
                 prefixArgs.Add("--verbosity");
-                prefixArgs.Add("Error");
+                prefixArgs.Add("minimal");
```

Comment:
Do we need prefixArgs at all?  

Also, I don't feel it is the CLI's place to decide what the "default" verbosity should be.  We should just pass through to NuGet, and NuGet gets to decide what the "default" verbosity is.